app/views/sign_up/select_email/show.html.erb

```diff
@@ -0,0 +1,43 @@
+<% self.title = t('titles.select_email') %>
+
+<%= render StatusPageComponent.new(status: :info, icon: :question) do |c| %>
+  <% c.with_header { t('titles.select_email') } %>
+  <p class="margin-top-4">
+    <%= I18n.t('help_text.select_preferred_email', sp: @sp_name, app_name: APP_NAME) %>
+  </p>
+
+  <%= simple_form_for('', url: sign_up_select_email_path) do |f| %>
+    <% @user_emails.each do |email, index| %>
+      <fieldset class="usa-fieldset">
```

Comment:
A `<fieldset>` is a grouping of several inputs, but we have this wrapping each individual input.

I think we also typically assign a name to fieldsets, usually using `aria-label` or `aria-labelledby`, though it doesn't seem to be a strict failure of any accessibility success criterion. (Related: [H71](https://www.w3.org/TR/WCAG20-TECHS/H71.html))